### PR TITLE
Add Google One Tap

### PR DESCRIPTION
**Pull request by nathan-stytch**
Add an example of SDK + Google One Tap

### CODE REVIEW

File: components/LoginEntryPoint.tsx [hunk 1]
```diff
@@ -24,6 +24,9 @@ const LoginEntryPoint = (props: Props) => {
       <button className={styles.entryButton} onClick={() => setLoginMethod(LoginMethod.EMAIL_WEBAUTHN)}>
         Email + WebAuthn (API Integration)
       </button>
+      <button className={styles.entryButton} onClick={() => setLoginMethod(LoginMethod.SDK_OAUTH)}>
+        OAuth + One Tap (SDK Integration)
+      </button>
```
Comment (chris-stytch) [inline components/LoginEntryPoint.tsx:29]: We'll need to clean up the login form, the height is no longer sufficient.

image

Comment (nathan-stytch) [reply components/LoginEntryPoint.tsx:29]: https://github.com/stytchauth/stytch-nextjs-integration/pull/47/commits/56ff7fc6a584a818e56680342278b45e96bbb430

**Review by chris-stytch — CHANGES_REQUESTED** [review]
Blocker:
- We need to adjust the README to include a note about One Tap, specifically that it is enabled by default and requires extra configuration if you'd like to try it out. Alternatively we could throw a few lines of text into the OAuth login form with some links to Docs. I think this second alternative is the best outcome but I'd consider a note in the README to resolve this block.

Non-blocker
- The height adjustment for the container exposes some jank in the SMS Passcodes form, i.e. the wordmark is now in the wrong place, I think we should try to fix if we can. If the problem isn't immediately obvious I'm ok with shipping as is and creating a ticket to fix, the same issue affects WebAuthn.

File: styles/Home.module.css [hunk 2]
```diff
@@ -31,26 +31,25 @@
...
-
 .primaryButton {
-  background-color: #19303D;
+  background-color: #19303d;
 }
 
 .primaryButton:disabled {
-  background-color: #F3F5F6;
-  color: #5C727D;
+  background-color: #f3f5f6;
```
Comment (chris-stytch) [inline styles/Home.module.css:43]: Out of curiosity, was this linting?

File: pages/index.tsx [hunk 3]
```diff
@@ -16,54 +17,25 @@ import LoginWithEmailWebAuthn from '../components/EmailWebAuthn/LoginWithEmail';
 //
 // VERCEL_URL only contains the domain of the site's URL, the scheme is not included so we must add it manually,
 // see https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables.
```
Comment (chris-stytch) [inline pages/index.tsx:19]: Please move this comment into `lib/getRedirectBaseUrl.ts`

Comment (nathan-stytch) [reply pages/index.tsx]: fixed https://github.com/stytchauth/stytch-nextjs-integration/pull/47/commits/e84494220b585655fc9847e0439b2aa8c9109d05

File: styles/Home.module.css (hunk 2 above)
Comment (nathan-stytch) [reply styles/Home.module.css:43]: i think so!

**Review by chris-stytch — APPROVED** [review]

**Pull request merged**